File: tournaments/alternatingSums/alternatingSums.cpp
std::vector<int> alternatingSums(std::vector<int> a) {
  int team1 = 0,
      team2 = 0;
  for (unsigned i = 0; i < a.size(); i += 2) {
    team1 += a[i];
  }
  for (unsigned i = 1; i < a.size(); i += 2) {
    team2 += a[i];
  }
  std::vector<int> result = {team1, team2};
  return result;
}

File: tournaments/alternatingSums/alternatingSums.js
function alternatingSums(a) {
    let teamA = 0;
    let teamB = 0;

    for (let i = 0; a.length > i; ++i) {
        if (i & 1) teamB += a[i];
        else teamA += a[i];
    }

    return [teamA, teamB];
}
